sdk/identity/Azure.Identity/src/AzureCliCredential.cs

```diff
@@ -157,9 +158,13 @@ private async ValueTask<AccessToken> RequestCliAccessTokenAsync(bool async, stri
                 Environment = { { "PATH", _path } }
             };
 
-        private static void GetFileNameAndArguments(string resource, out string fileName, out string argument)
+        private static void GetFileNameAndArguments(string resource, string tenantId, out string fileName, out string argument)
```

Comment:
These changes make me wonder if tenantId should be configurable when we construct the `AzureCliCredential` as well.

Reply:
Good point. I can add that.